mscore/scoretab.h

```diff
@@ -71,7 +71,7 @@ class ScoreTab : public QWidget {
       const TabScoreView* tabScoreView(int idx) const;
 
    signals:
-      void currentScoreViewChanged(ScoreView*);
+      void currentScoreViewChanged(Ms::ScoreView*);
```

Comment:
> Any idea why it crashes at the tst_runscripts mtest?

It looks like all `connect()` calls that use the old string-based connection syntax (with `SIGNAL()` and `SLOT()` macros) fail as they are unable to find signals by their old names. For example, for this signal there is the following warning in mtest log:
```
unknown:unknown: QObject::connect: No such signal Ms::ScoreTab::currentScoreViewChanged(ScoreView*) in /home/runner/work/MuseScore/MuseScore/mscore/musescore.cpp:6242
```

This may be the reason for failing script tests, as the log contains a lot of warnings which may result from signal-slot connections not working as expected.

Possible fixes are:
1) Move all `connect()` statements to the new syntax based on function pointers (https://wiki.qt.io/New_Signal_Slot_Syntax);
2) Change names of signals and slots in `connect()` statements to match these changes;
3) Revert these changes (but then the warnings would return back?)

Reply:
I'll revert those too, for now